client/src/components/generic/Header.js

```diff
@@ -128,43 +140,66 @@ export const Header = ({ hideEmployers = false }) => {
-                  text='Home'
-                  fullWidth={false}
-                  variant='outlined'
-                  color='inherit'
-                  onClick={() => {
-                    if (isParticipantPortal) {
-                      return history.push(Routes.ParticipantLanding);
-                    } else {
-                      return history.push(Routes.Admin);
-                    }
-                  }}
-                />
-              )}
-              {keycloak.authenticated && !keycloak.loginRequired ? (
-                <Button
-                  className={classes.button}
-                  text='Logout'
-                  fullWidth={false}
-                  variant='outlined'
-                  color='inherit'
-                  onClick={handleLogoutClick}
-                />
-              ) : (
-                <Button
-                  className={classes.button}
-                  text='Login'
-                  fullWidth={false}
-                  variant='outlined'
-                  color='inherit'
-                  onClick={handleLoginClick}
-                />
-              )}
-            </div>
+            <>
+              <Hidden xsDown>
+                <div className={classes.buttonWrapper}>
+                  {keycloak.authenticated && location.pathname !== Routes.Admin && (
+                    <Button
+                      className={classes.button}
+                      text='Home'
+                      fullWidth={false}
+                      variant='outlined'
+                      color='inherit'
+                      onClick={() => history.push(Routes.Admin)}
+                    />
+                  )}
+                  {keycloak.authenticated && !keycloak.loginRequired ? (
+                    <Button
+                      className={classes.button}
+                      text='Logout'
+                      fullWidth={false}
+                      variant='outlined'
+                      color='inherit'
+                      onClick={handleLogoutClick}
+                    />
+                  ) : (
+                    <Button
+                      className={classes.button}
+                      text='Login'
+                      fullWidth={false}
+                      variant='outlined'
+                      color='inherit'
+                      onClick={handleLoginClick}
+                    />
+                  )}
+                </div>
+              </Hidden>
+              <Hidden smUp>
+                <KeyboardArrowDownIcon onClick={handleMenuClick} />
+                <Menu
+                  id='login-menu'
+                  anchorEl={anchorEl}
+                  keepMounted
+                  open={Boolean(anchorEl)}
+                  onClose={handleMenuClose}
+                >
+                  {keycloak.authenticated && location.pathname !== Routes.Admin && (
+                    <MenuItem key='Home' onClick={() => history.push(Routes.Admin)}>
```

Comment:
In a case where a person is logged in in the participant portal, I think it makes sense to redirect them to the participant landing page. I added some of that function to my last PR, not sure what happened to it.

Reply:
I think it got lost in the merge, my bad.  I'll fix it up.